src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/product/form/save.js

```diff
@@ -58,6 +59,10 @@ define([
       this.showLoadingMask();
       this.getRoot().trigger('pim_enrich:form:entity:pre_save');
 
+      analytics.track('product-grid:product:saved', {
```

Comment:
The user is in the product edit form when he does this action

```suggestion
      analytics.track('product:form:saved', {
```